Source/Resource/Plugin/Language/en.csv

```diff
@@ -516,7 +516,7 @@ Genre_119;Freestyle
 Genre_120;Duet
 Genre_121;Punk Rock
 Genre_122;Drum Solo
-Genre_123;A Cappella
+Genre_123;A cappella

```

Comment:
In fact it should be `A capella`
http://id3.org/id3v2.3.0#Appendix_A_-_Genre_List_from_ID3v1

Reply:
Looks like the typo is on id3.org website.
[A cappella](https://en.wikipedia.org/wiki/A_cappella), with two "p".

Reply:
https://en.wikipedia.org/wiki/A_cappella

Reply:
Yes, it seem that nobody have notified them about that typo yet.
So as we cannot rely on what should be the reference...
https://en.wikipedia.org/wiki/ID3#standard

I do not agree on the capitalized letter as it seem to be the norm:
http://id3.org/id3v2.3.0#Appendix_A_-_Genre_List_from_ID3v1
https://en.wikipedia.org/wiki/ID3#standard

Reply:
I am in favor to keep the capitalized letter, as it is capitalized  in the "standard"

Reply:
so really what @JeromeMartinez  is saying is that it should remain `A Cappella`? Im consfused now.

What STANDARD are we following after all? This is all I want to know.

Reply:
@the-j0k3r Deep a little more and you will see that it is software that support officially the ID3 Tag standard, so they just apply what is in the standard.
http://id3.org/Implementations

Human been human, no blame can be done for following the standard or mistyping as we all do. Just looking at the IEC standards and all their edition addendum make it clear...

The standard ESPECIALLY should be followed.
As this genre in the ID3 standard, and many other ones, come from Winamp, "A Cappella" seem the correct choice.

We are just scratching the surface of what many people knew since very long time.

Reply:
Not digging any deeper, its a futile exercise and besides Mediainfo is actually more correct than that would be standard.

Reply:
Good 👍  it is now ONE more less than at start then ;)

And i'm sure you will correct much more than that later ;)

Reply:
This is the last of them, others already fixed in website and so on.

Reply:
Great, your english skill is much appreciated ;)

Reply:
English, hahahaha

Reply:
Hehe, as you saw we are not english native so your skill is welcome ;)

Reply:
Nor am I... :)

Reply:
Don't tell me that you learned english looking at CNN :)
Frenches and japaneses are known to have poor language skill, hopefully that is not your case ;)
You seem to be in far nordic land ;)